cmd/connectivity-registry/main.go

```diff
@@ -22,14 +22,21 @@ import (
 
 func main() {
 	var (
-		tlsCertPath string
-		tlsKeyPath  string
-		port        int
+		tlsCertPath                                   string
+		tlsKeyPath                                    string
+		port                                          int
+		orphanImportedServiceRecordDeleteDelaySeconds int
 	)
 
 	flag.StringVar(&tlsCertPath, "tls-cert", "", "the path to the server TLS certificate")
 	flag.StringVar(&tlsKeyPath, "tls-key", "", "the path to the server TLS key")
 	flag.IntVar(&port, "port", 8000, "the serving port for the hamlet server")
+	flag.IntVar(
+		&orphanImportedServiceRecordDeleteDelaySeconds,
+		"orphan-imported-service-record-delete-delay-seconds",
+		5 * 60,
+		"delay in seconds before an orphan imported service record is deleted"
```

Comment:
Is it worthwhile to describe why you would delay the deletion of the orphan service record here? It wouldn't be clear to me right off the bat why I would want to delay the deletion of orphaned records at all.

Reply:
we've updated the message